site/_posts/2017-11-06-call-me-gapri.md

```diff
@@ -0,0 +1,52 @@
+---
+title: "Last Week in ManageIQ: Please call me Gapri if it helps you relax"
+author: Roman Blanco
+date: 2017-11-07
+comments: true
+published: true
+tags: LWIMIQ
+---
+
+Wohoo! It's here again! All the features we were working on in last truly _Fine_ summer, are now released in new version, **Gaprindashvili**.
+
+[Roman](https://github.com/romanblanco) from UI team here, with my second LWIMIQ article. Lets see what happened last week.
+
+## Improved
+
+There is no week where we wouldn't improve at least a small thing :)
+Last week [Bill](https://github.com/bzwei) has improved [Ansible playbook logging](https://github.com/ManageIQ/manageiq/pull/16333), [Piotr](https://github.com/pkliczewski) has [made the Host trigger refresh after being removed](https://github.com/ManageIQ/manageiq-providers-ovirt/pull/127) and [Shmuel](https://github.com/smelamud) has [extended dialog to select VM by tag](https://github.com/ManageIQ/manageiq-providers-ovirt/pull/115)
+
+## Fixed
+
+With new features may also unfortunately come bugs. Last week [Adam](https://github.com/agrare) has helped ManageIQ to be less buggy by [fixing SCVMM credential validation](https://github.com/ManageIQ/manageiq-ui-classic/pull/2575), [David](https://github.com/ManageIQ/manageiq-ui-classic/pull/2573) updated GTL lists to [use custom pictures](https://github.com/ManageIQ/manageiq-ui-classic/pull/2573) and [Drew](https://github.com/d-m-u) fixed [Custom buttons with subclasses](https://github.com/ManageIQ/manageiq/pull/16378)
```

Comment:
Did you mean to instead use David's gitter profile?

Reply:
Sure, thanks 😄